distribution/lib/Standard/Table/0.0.0-dev/src/Data/Aggregate_Column.enso

```diff
@@ -77,19 +77,22 @@ type Aggregate_Column
           not missing value returned.
         - order_by: required for database tables. Specifies how to order the
           results within the group.
-    type Last (column:Column|Text|Integer) (name:Text|Nothing=Nothing) (ignore_nothing:Boolean=False) (order_by:Column_Selector|Nothing=Nothing)
+    type Last (column:Column|Text|Integer) (name:Text|Nothing=Nothing) (ignore_nothing:Boolean=True) (order_by:Column_Selector|Nothing=Nothing)
 
     ## Gets a column name to use for the aggregate column
     column_name : Table->Text
     column_name table =
         if this.name.is_nothing.not then this.name else
-            get_name c = (here.resolve_column table c).name
+            get_name c = (this.resolve_column table c).java_column.getName
```

Comment:
Why not use `.name`??

The naming logic should be shared between backends, but this method will not work with DB columns which do not have a `java_column`. But they do have a `name`.